docs/datamodel/constraints.rst

```diff
@@ -318,6 +318,97 @@ using arbitrary EdgeQL expressions. The example below uses the built-in
     );
   }
 
+
+Constraints and object inheritence
```

Comment:
Maybe Constraints and type inheritance? I see that term more and object inheritance feels more like a programming language as opposed to query language thing.